cumulus/parachains/runtimes/assets/asset-hub-rococo/src/xcm_config.rs

```diff
@@ -559,6 +562,8 @@ impl xcm_executor::Config for XcmConfig {
 	type IsReserve = (
 		bridging::to_westend::IsTrustedBridgedReserveLocationForConcreteAsset,
 		bridging::to_ethereum::IsTrustedBridgedReserveLocationForForeignAsset,
+		bridging::to_ethereum::IsTrustedBridgedReserveLocationForFeeAsset,
+		bridging::to_ethereum::IsTrustedBridgedReserveLocationForFeeAssetFromChildSovereign,
```

Comment:
Set Ethereum(with account as child sovereign) as the trusted reserve location for the FeeToken(i.e. DOT).